lib/rtcpeerconnection/edge.js

```diff
@@ -77,6 +122,112 @@ EdgeRTCPeerConnection.prototype.close = function close() {
   });
 };
 
+// NOTE(syerrapragada): Rollback implementation copied from chrome shim
+EdgeRTCPeerConnection.prototype.addIceCandidate = function addIceCandidate(candidate) {
+  var self = this;
+  if (this.signalingState === 'have-remote-offer') {
```

Comment:
@syerrapragada This can be:
```js
return this.signalingState === 'have-remote-offer'
  ? this._signalingStateLatch.when('low').then(...)
  : this._peerConnection.addIceCandidate(candidate);
```